assets/pages/paper-subscription-landing/components/form.jsx

```diff
@@ -0,0 +1,58 @@
+// @flow
+
+// ----- Imports ----- //
+
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+
+import { type PaperBillingPeriod } from 'helpers/subscriptions';
+import { type Action } from 'components/productPage/productPagePeriodForm/productPagePeriodFormActions';
+import ProductPagePeriodForm, { type StatePropTypes, type DispatchPropTypes } from 'components/productPage/productPagePeriodForm/productPagePeriodForm';
+
+import { type State } from '../paperSubscriptionLandingPageReducer';
+import { redirectToWeeklyPage, setPeriod } from '../paperSubscriptionLandingPageActions';
+
+
+// ---- Periods ----- //
+
+const billingPeriods = {
```

Comment:
Would be amazing if this was a json blob in the repo that we could open up for editing by non-engineers (creating a mini cms). Out of scope for this PR though.